rafthttp/http.go

```diff
@@ -171,25 +170,25 @@ func (h *streamHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	fromStr := path.Base(r.URL.Path)
 	from, err := types.IDFromString(fromStr)
 	if err != nil {
-		log.Printf("rafthttp: failed to parse from %s into ID", fromStr)
+		plog.Errorf("failed to parse from %s into ID (%v)", fromStr, err)
 		http.Error(w, "invalid from", http.StatusNotFound)
 		return
 	}
 	if h.r.IsIDRemoved(uint64(from)) {
-		log.Printf("rafthttp: reject the stream from peer %s since it was removed", from)
+		plog.Warningf("rejected the stream from peer %s since it was removed", from)
 		http.Error(w, "removed member", http.StatusGone)
 		return
 	}
 	p := h.peerGetter.Get(from)
 	if p == nil {
-		log.Printf("rafthttp: fail to find sender %s", from)
+		plog.Warningf("failed to find sender %s", from)
```

Comment:
ideally we need to add a comment for why this would happen and why this is an error case